{bio}[GCCcore/11.3.0] PSORTb v3.0.6, libpsortb v1.0

bedroge opened the pull request:
(created using `eb --new-pr`)

bedroge commented:
@boegelbot please test @ generoso

bedroge commented:
Test report by @bedroge
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
interactive2 - Linux Rocky Linux 8.7, x86_64, AMD EPYC-Milan Processor, Python 3.6.8
See https://gist.github.com/bedroge/448e69250ba404fdda3c6498d2f4fbd9 for a full test report.

verdurin commented:
Test report by @verdurin
**FAILED**
Build succeeded for 2 out of 3 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/9f8001cc5fd0c9f04038111dc218103a for a full test report.

bedroge commented:
> ```
> Is this not supported on EL7?

I don't know, but it looks like it somehow may have picked up stuff from the host?

According to the user who requested this, some Perl modules were missing. This was probably due to the newer version of BioPerl, which split off several things to separate modules. I've added a bunch of dependencies, and also converted this to a `PerlBundle` easyblock, which requires https://github.com/easybuilders/easybuild-easyblocks/pull/2945.

bedroge commented:
Test report by @bedroge
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2945
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.6
See https://gist.github.com/bedroge/4e4ba8c1db82f5ca6bd80fbce7f089d5 for a full test report.

bedroge commented:
Test report by @bedroge
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2945
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
interactive2.hb.hpc.rug.nl - Linux Rocky Linux 8.7, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/762843fee055da2ab6b224121f8294db for a full test report.

bedroge commented:
@boegelbot please test @ generoso

bedroge commented:
@boegelbot please test @ jsc-zen2

bedroge commented:
@boegelbot please test @ generoso
